helm-chart/templates/secrets/storage.yaml

```diff
@@ -1,11 +1,13 @@
-{{ if and (not .Values.storage.deploy) (ne .Values.storage.driver "InMemory") }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: {{ include "miniurl.storageSecretName" . }}
 type: Opaque
 stringData:
 {{ if eq .Values.storage.driver "Postgres" }}
+{{ if and (not .Values.storage.deploy) (ne .Values.storage.driver "InMemory") }}
   password: {{ required "storage.postgresConfig.secrets.password is required if the storage.driver is set to 'Postgres' and storage.deploy is set to false" }}
 {{ end }}
-{{ end }}
+data:
+  postgresql-password: {{ randAlphaNum 20 | b64enc }}
```

Comment:
There are other occurences referencing this password in the Helm chart. Can you review those places and replace them accordingly.

##### Hints
* Check all `miniurl.storageSecretName` uses
* Check all `.Values.storage.deploy` uses
* See templates/deployment.yaml lines 46-50 for an example